connections/src/app/core/services/qr-code-generator.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class QrCodeGeneratorService {
+
+  constructor() { }
+
+  
+  generateQrCode(data: string): string {
+    // Simulate an endpoint that returns a QR code URL
+    // In a real application, you would make an HTTP request to your backend here
+    return `https://api.com/v1/create-qr-code/?data=${encodeURIComponent(data)}`;
```

Comment:
@Siddharth-Tiwari1712 Tell me what we need to back-end to have via an API. We basically get to design what we need

Reply:
This was just a sample so that we get unique string.
I'll update you on api